src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -81,17 +81,22 @@ trait IterableOps[+A] extends Any {
   /** A view representing the elements of this collection. */
   def view: View[A] = View.fromIterator(iterator())
 
-  /** Given a collection factory `fi` for collections of type constructor `C`,
-    *  convert this collection to one of type `C[A]`. Example uses:
+  /** Given a collection factory `fi`, convert this collection to the appropriate
+    * representation for the current element type `A`. Example uses:
     *
     *      xs.to(List)
     *      xs.to(ArrayBuffer)
+    *      xs.to(BitSet) // for xs: Iterable[Int]
     */
-  def to[C[X] <: Iterable[X]](fi: FromIterable[C]): C[A @uncheckedVariance] =
-  // variance seems sound because `to` could just as well have been added
-  // as a decorator. We should investigate this further to be sure.
+  def to[F <: TypeConstrainedFromIterable[A]](fi: F): fi.To[A @uncheckedVariance] =
     fi.fromIterable(coll)
 
+  // Generic version of the method above that can build anything with BuildFrom. Note that `bf` is not implicit.
+  // We never want it to be inferred (because a collection could only rebuild itself that way) but we do rely on
+  // the implicit conversions from the various factory types to BuildFrom.
+  def to(bf: BuildFrom[Iterable[A], A]): bf.To =
+    bf.fromIterable(coll)(coll)
+
   /** Convert collection to array. */
```

Comment:
Hm, actually `BitSet` doesn't currently work with the `BuildFrom` version. I'll try to fix it.